### PR TITLE
Libary updates to latest

### PR DESCRIPTION
**Pull request by matzew**
@odra @david-martin

updates to latest versions, due to security issues:
https://snyk.io/test/npm/fh-mbaas-express

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.
Fhcap PR will be generated automatically.
To deploy your changes to a cluster place comment in this PR with pattern [x][y] where x=deploy and y=clustername you wish to deploy. 
You can also deploy your change manually using fhcap-cli

File: lib/cloud/cloud.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var express = require('express');
+	var express = require('express');
```
Comment (fh-qube) [inline lib/cloud/cloud.js]: MINOR Replace all tab characters in this file by sequences of white-spaces. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATabCharacter)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by matzew** [comment]
All up to date and build passing

@david-martin any comments ?

**Comment by david-martin** [comment]
@matzew  @odra this looks good.
Nice to have it building openly on travis :+1:

**Pull request merged**